### ISSUE TITLE
Roadmap to TSERS 2.0

### ISSUE THREAD

**Issue by ghost** [tsers-js/core#18]
It would be nice if we had a roadmap to TSERS 2.0. I'm looking forward the next release!

**Comment by milankinen** [comment]
Yes, I should definitely do the official release roadmap as soon as I manage to get some open source time!

Unofficially, 2.0 release will make the TSERS core and drivers streaming library agnostic and contain some minor bug fixes. But that's basically all for 2.0. See new API proposal (and Kefir usage) **[here](https://github.com/tsers-js/examples/blob/2.x/examples/kefir.js)**.

Until I manage to write an official roadmap, the progress of 2.0 release can be followed from issue #5.
